jenkins/opensearch/smoke-test.jenkinsfile

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+lib = library(identifier: 'jenkins@8.1.0', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+def docker_images = [
+    'tar': 'opensearchstaging/ci-runner:ci-runner-al2-opensearch-build-v1',
+    'rpm': 'opensearchstaging/ci-runner:ci-runner-almalinux8-systemd-base-integtest-v1',
+    'deb': 'opensearchstaging/ci-runner:ci-runner-ubuntu2004-systemd-base-integtest-v3',
+    'zip': 'opensearchstaging/ci-runner:ci-runner-windows2019-opensearch-build-v1',
+]
```

Comment:
Looks like we need to add these docker images sooner as workflows are increasing. https://github.com/opensearch-project/opensearch-build/issues/4233